applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -195,6 +195,15 @@ config DESKTOP_BLE_SCAN_KEYBOARD_LIMIT
 	range 0 255
 	depends on DESKTOP_BLE_SCANNING_ENABLE
 
+config DESKTOP_BLE_SCAN_PM_EVENTS
```

Comment:
We already use `_PM_EVENTS` suffix in all of our CAF modules to trigger handling power management events. Maybe we can use it here too for consistency? See ble_adv, buttons, leds and click_detector

Reply:
ok - let's keep it as is. But the name is wrong since the first thing it brought to my mind is tat it enable some events. It could be EVENTS_SUPPORT event. Why have we named it so badly...